src/api/api.py

```diff
@@ -0,0 +1,34 @@
+from .pros import pros
+from flask import Flask, jsonify, abort
+from werkzeug.exceptions import HTTPException
+from flask_cors import CORS
+
+cors = CORS()
+
+api = Flask(__name__)
+
+cors.init_app(api)
+
+#error handlers
+@api.errorhandler(HTTPException)
+def http_error(error):
+    """
+    Handles all HTTP errors
+    """
+    return jsonify({'error': str(error)}), error.code
```

Comment:
Inline dictionaries should enclose a space on each side of the brace. For eg: `{ 'error': 'Nuclear explosion' }`.